### PR TITLE
Translated a bunch of strings

### PR DESCRIPTION
**Pull request by springstan**
Issues affected:  #3429

Translated:
- customization page
- device table
- device config page
- multiple notes in different files (dialog-entity-registry-detail.ts, ha-config-entry-page.ts and ha-logbook.js)
- confirmation message for custom panels

Results: (showing all translations even errors)
customize
device
devices_table

### CODE REVIEW

File: src/panels/custom/ha-panel-custom.ts [hunk 1]
```diff
@@ -78,13 +78,23 @@ export class HaPanelCustom extends UpdatingElement {
...
+            tempA.href
+          ) +
+            "\n\n" +
+            this.hass.localize(
+              "ui.panel.custom.external_panel.complete_access"
+            ) +
+            "\n\n" +
+            "(" +
+            this.hass.localize("ui.panel.custom.external_panel.hide_message") +
+            ")"
```
Comment (bramkragten) [inline src/panels/custom/ha-panel-custom.ts]: Use a template literal instead of concatting strings.

File: src/translations/en.json [hunk 2]
```diff
@@ -726,7 +726,19 @@
           "picker": {
             "header": "Customization",
             "introduction": "Tweak per-entity attributes. Added/edited customizations will take effect immediately. Removed customizations will take effect when the entity is updated."
-          }
+          },
+          "warning": {
+            "include_sentence": "It seems that your configuration.yaml doesn't properly",
+            "include_link": "include customize.yaml",
```
Comment (bramkragten) [inline src/translations/en.json]: We should bring include to the other key and include the link as a variable in that string.

Comment (springstan) [reply src/translations/en.json]: I am not quite sure what you mean by that. Do you want to pass the entire `<a>` into one translation string?

File: src/panels/custom/ha-panel-custom.ts (hunk 1 above)
Comment (springstan) [reply src/panels/custom/ha-panel-custom.ts]: Resolved: https://github.com/home-assistant/home-assistant-polymer/pull/4137/commits/4b6810be37ef69730daed9a9ffb1c4046b2be5a2

File: src/translations/en.json (hunk 2 above)
Comment (bramkragten) [reply src/translations/en.json]: Yes

Comment (springstan) [reply src/translations/en.json]: Like this?
```json
"include_sentence": "It seems that your configuration.yaml doesn't properly {link}"
```

Comment (bramkragten) [reply src/translations/en.json]: I would do:
`"include_sentence": "It seems that your configuration.yaml doesn't properly include {customize_link}"`

Comment (springstan) [reply src/translations/en.json]: Oddly enough, I cannot pass the entire `<a>` into the string:
```js
[[localize('ui.panel.config.customize.warning.include_sentence',
          'customize_link',
          "<a
          href="https://www.home-assistant.io/docs/configuration/customizing-devices/#customization-using-the-ui"
          target="_blank"
          >configuration.yaml</a>")]]
```
Just prints it out like that as the innerHTML of the `div` element. 
Does the `localize` funciton filter html elements?

Comment (bramkragten) [reply src/translations/en.json]: yeah probably... OK, never mind what I said.

Comment (springstan) [reply src/translations/en.json]: So how should I address this? Just leave it like that or is there any other option?

File: src/panels/config/devices/ha-config-device-page.ts [hunk 3]
```diff
@@ -122,7 +122,11 @@ export class HaConfigDevicePage extends LitElement {
 
     if (!device) {
       return html`
-        <hass-error-screen error="Device not found."></hass-error-screen>
+        <hass-error-screen
+          error="${this.hass.localize(
```
Comment (bramkragten) [inline src/panels/config/devices/ha-config-device-page.ts]: If a value is dynamic you can drop the `"` around them.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**